mne/stats/regression.py

```diff
@@ -134,3 +142,197 @@ def _fit_lm(data, design_matrix, names):
         mlog10_p_val[predictor] = -np.log10(p_val[predictor])
 
     return beta, stderr, t_val, p_val, mlog10_p_val
+
```

Comment:
NJ Smith discusses what output is best for continuous regression in the papers extensively and argues coefs are best. Keep in mind you can easily make this pretty much r values by standardizing your raw data first!